scripts/windows/InstallScript.ps1

```diff
@@ -30,8 +24,8 @@ if ([string]::IsNullOrEmpty($Token))
     Exit
 }
 
-$source = "http://static.zorustech.com.s3.amazonaws.com/downloads/ZorusInstaller.exe";
-$destination = "$env:TEMP\ZorusInstaller.exe";
+$source = "http://static.zorustech.com.s3.amazonaws.com/downloads/ZorusInstaller.exe"

```

Comment:
use s3 rather than cloudfront

Reply:
Let's talk about this before we merge this in please.